requirements.txt

```diff
@@ -2,3 +2,4 @@ futures;python_version<="2.7"
 python-dateutil
 pyopenssl
 msrestazure>=0.4.0,<0.5.0
+vcrpy
```

Comment:
Remove that, vcrpy is installed by the test requirements file (see .travis.yaml)